fedimint-server/src/consensus/server.rs

```diff
@@ -213,13 +216,19 @@ impl ConsensusServer {
         let client_cfg = cfg.consensus.to_config_response(&module_inits);
         let modules = ModuleRegistry::from(modules);
 
+        let rejoin_at_epoch: RejoinAtEpoch = Default::default();
         let consensus_api = ConsensusApi {
             cfg: cfg.clone(),
             db: db.clone(),
             modules: modules.clone(),
             client_cfg,
             api_sender,
             supported_api_versions: ServerConfig::supported_api_versions_summary(&modules),
+            rejoin_at_epoch: Arc::clone(&rejoin_at_epoch),
+            peer_status_channels,
+            // keep the status for a short time to protect the system against a denial-of-service
+            // attack
+            full_status_cache: ExpiringCache::new(Duration::from_secs(1)),
```

Comment:
What's the DoS attack here?  Generally we don't prevent these here, we will have to eventually deal with them at the consensus layer.

Reply:
Someone may call this unauthenticated endpoint a lot of times, causing a slowdown in the application. A similar concern exist for similar endpoints like `fetch_epoch_count`.

Reply:
`fetch_epoch_count` is probably trivially fast. In this case a message is put on the queue to all the peer threads, so there's (some, probably minor) a possibility that someone could cause these to be always full and stale the communication between consensus thread and peer threads.